backend/de.metas.contracts/src/main/java/de/metas/contracts/flatrate/process/C_Flatrate_Term_Change_For_FlatrateData.java

```diff
@@ -22,20 +22,31 @@
 
 package de.metas.contracts.flatrate.process;
 
+import com.google.common.collect.ImmutableList;
 import de.metas.contracts.FlatrateDataId;
 import de.metas.contracts.IFlatrateDAO;
+import de.metas.contracts.model.I_C_Flatrate_Data;
 import de.metas.contracts.model.I_C_Flatrate_Term;
+import de.metas.process.Param;
+import de.metas.process.ProcessInfoParameter;
 import de.metas.util.Services;
 
 public class C_Flatrate_Term_Change_For_FlatrateData extends C_Flatrate_Term_Change_Base
 {
 	final IFlatrateDAO flatrateDAO = Services.get(IFlatrateDAO.class);
 
+	public static final String PARAM_FLATRATE_DATA_ID = I_C_Flatrate_Data.COLUMNNAME_C_Flatrate_Data_ID;
+
+	@Param(parameterName = PARAM_FLATRATE_DATA_ID, mandatory = true)
+	private int p_flatrateDataId;
+
```

Comment:
same, here:

This will make the process unusable from flatratedata window.
If the process shall not be run from there, pls remove the process from that window.
If still shall be run from there, then, pls make the process work in both ways.

Reply:
the process works on both windows